common4j/src/main/com/microsoft/identity/common/java/providers/microsoft/microsoftsts/MicrosoftStsOAuth2Strategy.java

```diff
@@ -619,10 +622,23 @@ protected TokenResult getTokenResultFromHttpResponse(@NonNull final HttpResponse
                     tokenResponse.setCliTelemSubErrorCode(cliTelemInfo.getServerSubErrorCode());
                 }
             }
-
-            SpanExtension.current().setAttribute(
-                    AttributeName.ccs_request_id.name(),
-                    response.getHeaderValue(XMS_CCS_REQUEST_ID, 0));
+            final String ccsRequestId = response.getHeaderValue(XMS_CCS_REQUEST_ID, 0);
+            if(null != ccsRequestId){
```

Comment:
nit: space after if

Reply:
Thanks for catching this, I realized the style I was using wasn't in sync with the rest of the code (I wasn't using a space between if/for and the opening parens), so I fixed all the instances I wrote in my latest rev.